index.html

```diff
@@ -430,35 +415,33 @@ <h5 id="web-monetization-meta-tag">Web Monetization Meta Tag</h5>
           </pre
         >
       </section>
-      <section data-dfn-for="Monetization" data-link-for="Monetization">
+      <section data-dfn-for="Monetization">
         <h2><dfn>Monetization</dfn> interface</h2>
         <pre class="idl">
-              [Exposed=Window]
-              interface Monetization : EventTarget {
-                readonly attribute DOMString state;
-    
-                attribute EventHandler onmonetizationstart;
-                attribute EventHandler onmonetizationprogress;
-              };
-          </pre
-        >
+          [Exposed=Window, SecureContext]
+          interface Monetization : EventTarget {
+            readonly attribute MonetizationState state;
+            attribute EventHandler onstart;
```

Comment:
@marcoscaceres 
What does this `attribute  EventHandler onstart` actually mean ? 
`monetization.onstart = function() {}` as per `img.onload = function() {}` ?

The current implementation (and the example code [right here in the explainer](https://github.com/adrianhopebailie/web-monetization/blob/master/explainer.md#handle-payments)) is using events with all the data inside a `detail` object on the event.